docs/python/visualization/shape.rst

```diff
@@ -88,6 +89,36 @@ Shape
         mode = 2 
             displays schematic. ie line through 1st and last 2d points of each 
             section. 
+        .. code-block::
+            python
+
+            import plotly
+            from neuron import h, gui
+            from neuron.units import mV, ms
+            import matplotlib
+            h.load_file('c91662.ses')
+
+            for sec in h.allsec():
+                sec.nseg = int(1 + 2 * (sec.L // 40 ))
+                sec.insert(h.hh)
+
+            ic = h.IClamp(h.soma(0.5))
+            ic.delay = 1 * ms
+            ic.dur = 1 * ms
+            ic.amp = 10
+
+            h.finitialize(-65 * mV)
+            h.continuerun(2 * ms)
+
+            from matplotlib import cm
+            ps = h.PlotShape(False)
+            ps.variable("v")
+            print(ps.show()) # prints the current mode
+            ps.show(0) # alters the mode to 0 that displays diameters for each segment
+            print(ps.show())  # should print 0 as the mode set
+            ps.plot(plotly, width=7, cmap=matplotlib.colormaps["viridis"]).show()
+        
+    
 
```

Comment:
@sgamiye : @adamjhn suggested to format the code example. Would you prefer to change it?